ledger/src/blockstore_db.rs

```diff
@@ -955,6 +1009,17 @@ impl Default for BlockstoreOptions {
             access_type: AccessType::PrimaryOnly,
             recovery_mode: None,
             enforce_ulimit_nofile: true,
+            shred_storage_type: ShredStorageType::RocksLevel,
+            // Maximum size of cf::DataShred.  Used when `shred_storage_type`
+            // is set to ShredStorageType::RocksFifo.  The default value is set
+            // to 125GB, assuming 500GB total storage for ledger where 25% is
+            // used by data shreds.
+            shred_data_cf_size: 125 * 1024 * 1024 * 1024,
```

Comment:
Updated the default value based on the suggested 500GB minimum ledger storage requirement.

Reply:
nit: move `125` to a `const MAX_FIFO_COMPACTION_DATA_CF_SIZE`